services/net/syndication/SyndicationAction.cs

```diff
@@ -143,11 +143,19 @@ private async Task FetchContent(IngestModel ingest, SyndicationItem item, Uri? l
             if (link != null && Uri.IsWellFormedUriString(link.ToString(), UriKind.Absolute))
             {
                 var html = await this.GetContentAsync(link);
-                if (item.PublishDate == DateTime.MinValue) item.PublishDate = GetPubDateTime(html, ingest);
+                if (item.PublishDate.Date == DateTime.MinValue) item.PublishDate = GetPubDateTime(html, ingest);
                 var text = StringExtensions.SanitizeContent(html);
+                if (text.StartsWith("By "))
+                {
+                    var separator = "\n\n";
+                    var authorName = text.Split(separator)[0];
```

Comment:
Won't this result in taking the first line of the text "By author, and any other text on the line"?

Reply:
We only want the author's name

Reply:
Yes this will take the first line of the text, in most cases these would be the author's name(s), however in some cases title could be included as well, such as the example as mentioned in the ticket (https://apps.itsm.gov.bc.ca/jira/browse/TNO-1018). Thanks